### PR TITLE
changelog and strings for 13.0.1, 12.0.6 and 11.0.8

### PR DESCRIPTION
**Pull request by jospoortvliet**
Done with the nice script from @MorrisJobke 

:+1: 

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@Jurrie I also saw the 13 link fix, after you of course... perhaps you can have a quick look here to see what I missed ;-)

File: page-changelog.php [hunk 1]
```diff
@@ -4,14 +4,101 @@
...
+    <li><a href="https://github.com/nextcloud/files_texteditor/pulls/83">Fix ACE module files failing to load (files_texteditor#83)</a></li>
+    <li><a href="https://github.com/nextcloud/files_texteditor/pulls/89">Do not keep FileInfoModels returned by "getModelForFile" (files_texteditor#89)</a></li>
+    <li><a href="https://github.com/nextcloud/gallery/pulls/394">Fix share drop down in gallery not properly shown (gallery#394)</a></li>
+    <li><a href="https://github.com/nextcloud/gallery/pulls/402">Fix gallery button hidden in folders without create permission (gallery#402)</a></li>
+    <li><a href="https://github.com/nextcloud/gallery/pulls/405">Fix controls position in gallery layout (gallery#405)</a></li>
+    <li><a href="https://github.com/nextcloud/gallery/pulls/406">Removed old code already present thanks to the files app (gallery#406)</a></li>
+    <li><a href="https://github.com/nextcloud/gallery/pulls/407">Create "file app" public share links if the slideshow is opened from ... (gallery#407)</a></li>
+    <li><a href="https://github.com/nextcloud/password_policy/pulls/61">Allow to check against haveibeenpwned.com password list (password_policy#61)</a></li>
```
Comment (MorrisJobke) [inline page-changelog.php]: @jospoortvliet Up to you if you want to put all the app related stuff into a separate list or not.

Comment (jospoortvliet) [reply page-changelog.php]: This is fine as-is.

**Comment by MorrisJobke** [comment]
I also updated the dates.

**Pull request merged**